packages/eslint-plugin/src/rules/no-empty-styled-expression/index.ts

```diff
@@ -25,9 +25,20 @@ const createNoEmptyStyledExpressionRule =
     return {
       'CallExpression[callee.type="MemberExpression"]': (node: CallExpression) => {
         const { references } = context.getScope();
-        const isStyledImported = references.some((reference) =>
-          reference.resolved?.defs.some(isStyledImportSpecifier)
-        );
+
+        if (node.callee.type !== 'MemberExpression') {
+          return;
+        }
+
+        const membEx = node.callee;
```

Comment:
I'd just call it callee via `const { callee } = node;` or make it longhand `const { callee: memberExpression } = node;`